org.eclipse.buildship.core/src/main/resources/org/eclipse/buildship/core/i18n/CoreMessages.properties

```diff
@@ -49,4 +49,6 @@ ErrorMessage_0_MustBeDirectory={0} must be a directory.
 ErrorMessage_0_AlreadyExists={0} already exists.
 
 Preference_Label_GradleUserHome=Gradle User Home
-Preference_Label_OfflineMode=Offline mode
\ No newline at end of file
+Preference_Label_OfflineMode=Offline mode
+Preference_Label_BuildScans=Enable Build Scans
+Preference_Label_BuildScansHover=If this option is enabled then all projects that apply the build scans plugin will publish a build can upon each Gradle invocation.
```

Comment:
Done. One comment: if we want to make links rendered and clickable we need to invest more time in the hover widget.